Fix technology table

cuzuco2 opened the pull request:
Please review):

Fix #71

Test URLs:
- Before: https://main--elco-aerotop-sg--hlxsites.hlx.page/de/
- After: https://fix-technology-table--elco-aerotop-sg--hlxsites.hlx.page/de/

Commit:
fixed several syntax issues following the comments on technology table

Commit:
Fixed js to avoid one foreach and replace it with element.child

Commit:
Fixed code to avoid unnecesary loops

solaris007 commented on blocks/technology-table/technology-table.js:
```diff
@@ -1,25 +1,26 @@
 function buildTechnologyTable(block) {
```
@cuzuco2 the piece that is missing is that you can source an array from the node list and therefore use all the nice array functions. i'd re-write the function like this:

```
function buildTechnologyTable(block) {
  const rows = block.querySelectorAll('.technology-table > div');
  rows.forEach((cell, i) => {
    Array.from(cell.children)
      .forEach((child) => {
        if (i === 0) {
          const titleText = child.textContent;
          const title = document.createElement('h1');

          child.textContent = '';
          title.textContent = titleText;
          title.style.fontSize = 'var(--heading-font-size-l)';
          title.style.textAlign = 'left';

          child.appendChild(title);
          child.classList.add('techno-table-title');
        } else if (i === 1) {
          child.classList.add('techno-table-image');
        } else if (i === 2) {
          child.classList.add('techno-table-description');
        }
      });
  });
}
```

cuzuco2 replied:
Ah ok that was it, thanks

solaris007 committed:
Merge branch 'main' into fix-technology-table

Commit:
Changes htmlcollection to array

Commit:
Getting remore content
Merge branch 'fix-technology-table' of github.com:hlxsites/elco-aerotop-sg into fix-technology-table

cuzuco2 commented:
I just discovered that the text is not aligned correctly and should be aligned left.
I will do that and resubmit the PR

Commit:
Fixed issues of alignement and externalised css from javascript to css

Commit:
Fixed some changes on js
